### PR TITLE
Fixing issue where the dashboard app didn't have reference to the states API

### PR DESCRIPTION
**Pull request by dborgen2**
## What’s changing?

The environment variable for the states API was missing from the IAC. This caused an issue where the states weren't getting populated on the Metrics dashboard. It worked locally because the environment variable is set on my machine, and hadn't taken into account the IAC.

This fixes Line 13 and Line 18 from the UX QA Findings spreadsheet

## Why?

Closes NAC-1488

## This PR has:

- [ ] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [x] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [ ] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [x] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

File: iac/create-metrics-resources.bash [hunk 1]
```diff
@@ -390,7 +405,9 @@ main () {
       diagnosticSettingName="${DIAGNOSTIC_SETTINGS_NAME}" \
       eventHubAuthorizationRuleId="${EH_RULE_ID}" \
       eventHubName="${EVENT_HUB_NAME}" \
-      workspaceId="${LOG_ANALYTICS_WORKSPACE_ID}"
+      workspaceId="${LOG_ANALYTICS_WORKSPACE_ID}" \
+      StatesApiUri="$states_api_uri" \
+      StatesApiAppId="$states_api_app_id" \
```
Comment (bgfoshay) [inline iac/create-metrics-resources.bash:410]: Nitpick: you don't need the trailing slash

**Review by bgfoshay — APPROVED** [review]
👍

File: iac/arm-templates/dashboard-app.json [hunk 2]
```diff
@@ -68,6 +68,12 @@
             "metadata": {
                 "description": "Specify the Log Analytics Workspace Id."
             }
+        },
+        "StatesApiUri": {
```
Comment (bgfoshay) [inline iac/arm-templates/dashboard-app.json:72]: Nitpick: should the parameter names be lowercase?

Comment (dborgen2) [reply iac/arm-templates/dashboard-app.json]: Yea, I fixed it to be consistent within this ARM template. The Query Tool template that I copied it from there's a mix of upper/lower case for some reason

**Pull request merged**